src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -163,7 +169,14 @@ public void scenario1() throws IOException {
 
         logger.debug("Can username 'smith123' read " + testObj);
         setAuth(request, "smith123");
-        assertEquals(HttpStatus.SC_OK, getStatus(request));
+        try (final CloseableHttpResponse response = execute(request)) {
+            assertEquals(HttpStatus.SC_OK, getStatus(response));
+
+            final Optional<String> header = Arrays.asList(response.getHeaders("Link")).stream().map(Header::getValue)
+                    .filter(aclLink::equals).findFirst();
+            assertTrue("Missing Link header", header.isPresent());
```

Comment:
I got this from @acoburn, so he can confirm.
But if I read it correctly it filters for the above created Link. So if that Link doesn't exist this would fail. It shouldn't check for "any" Link header.